### PR TITLE
chore(stdlib): Replace most `@disableGC` annotations with `@unsafe`

### PR DESCRIPTION
**Pull request by jozanza**
`@unsafe` is bussin'

### CODE REVIEW

File: stdlib/array.gr [hunk 1]
```diff
@@ -12,34 +12,28 @@ import WasmI32 from "runtime/unsafe/wasmi32"
 import Memory from "runtime/unsafe/memory"
 import { allocateArray, tagSimpleNumber } from "runtime/dataStructures"
 import Exception from "runtime/exception"
+import { coerceNumberToWasmI32 } from "runtime/numbers"
+// import PrintWasm from "runtime/unsafe/printWasm"
```
Comment (ospencer) [inline stdlib/array.gr:16]: comment

File: stdlib/array.gr [hunk 2]
```diff
@@ -51,13 +45,10 @@ let initLength = length => {
  * @example Array.length([> 1, 2, 3, 4, 5]) == 5
  * @since v0.1.0
  */
-@disableGC
+@unsafe
 export let rec length = array => {
```
Comment (ospencer) [inline stdlib/array.gr:49]: not recursive

**Review by phated — APPROVED** [review]
Looks great! And the tests are green now 🎉  Thanks @jozanza!

**Review by ospencer — APPROVED** [review]
Thanks @jozanza!

**Pull request merged**